Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+/* eslint-env node */
+"use strict";
+
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        // Project package file destination.
+        pkg: grunt.file.readJSON("package.json"),
+        eslint: {
+            all: ["**/*.js"]
+        },
+        jsonlint: {
+            all: ["*.json", ".*.json"]
+        }
+    });
+
+    // Load the plugin(s):
+    grunt.loadNpmTasks("fluid-grunt-eslint");
```

Comment:
This project doesn't contain any .js files and we imagine never well - the eslint tasks directed at the project itself should be removed